lua/package-info/utils/register-autocmd.lua

```diff
@@ -1,11 +1,13 @@
 local constants = require("package-info.utils.constants")
+local group = vim.api.nvim_create_augroup(constants.AUTOGROUP, {clear = true})
 
 --- Register given command when the event fires
 -- @param event: string - event that will trigger the autocommand
 -- @param command: string - command to fire when the event is triggered
-return function(event, command)
-    vim.cmd("augroup " .. constants.AUTOGROUP)
-    vim.cmd("autocmd!")
-    vim.cmd("autocmd " .. event .. " * " .. command)
-    vim.cmd("augroup end")
+return function(event, pattern, command)
```

Comment:
Can you please add the doc above for the new param?

Reply:
Thanks. Is that a specific format that is recognized by LSP? The previous format was just me adding it how it looked readable. If this follows some format and is recognized by LPS, ill probably use them in the whole plugin.

Reply:
I use `coc.nvim` with `coc-sumneko-lua` and it recognizes them. I also don't think that there is supposed to be a colon in there anywhere. I just moved it over after the type because it ignores the colon whenever doing a definition hover if it is in that place.

So for example, the following is how I have seen them in most of the source code I have looked at. No colon anywhere and no capital letters. Though, using a colon where I placed it and/or using a capital letter after the type most definitely helps for readability.
```lua
---@param event string event that will trigger the autocommand
```

The [`emmylua` documentation](https://emmylua.github.io/annotation.html) isn't the greatest either.